src/rank_llm/demo/rerank_dataset_with_custom_index.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+parent = os.path.dirname(SCRIPT_DIR)
+parent = os.path.dirname(parent)
+sys.path.append(parent)
+
+from rank_llm.retrieve.retriever import Retriever
+from rank_llm.rerank.vicuna_reranker import VicunaReranker
+
+"""
+Data prep: download the pyserini prebuilt index and topics files into any directory:
```

Comment:
I want the demo to be executable as is. You can run bash in python scripts, please use it to get the data into a path and use it with pyserini Retriever

Reply:
sure, this can be done... the index size is 60mb so I was hesitant to just put a command in there to download it but yes I'll make that change

Reply:
change made